src/components/Auth.vue

```diff
@@ -198,56 +203,86 @@ export default {
     postSecure: function (link, params) {
       return this.doSecure(axios.post, link, params);
     },
+    retrieveRoad: function (roadID) {
+      const _this = this;
+      this.gettingUserData = true;
+      return this.getSecure('/sync/roads/?id=' + roadID).then(function (roadData) {
+        if (roadData.status === 200 && roadData.data.success) {
+          roadData.data.file.downloaded = moment().format(DATE_FORMAT);
+          roadData.data.file.changed = moment().format(DATE_FORMAT);
```

Comment:
It should probably depend on the error. If it's an outdated token, then it should log the user out. If it's some other error maybe it should try again? For now we can maybe add back in the logout

Reply:
Actually, it looks like it's still doing the logout on error since the error would be caught within getUserData.

Reply:
it'll never show this helpful alert though, right? because it would be a type error or undefined or something:
```js
          if (err === 'Token not valid') {
            alert('Your token has expired.  Please log in again.');
          }
```

we definitely want an error if it's not a success right? let's make it explicit

Reply:
verify will return that error if the token is expired

Reply:
oh gotcha, so there isn't really a world in which it ever reaches that if statement if the statement would be false?

Reply:
I think there might be cases where response.data.success isn't true or the request doesn't go through or something where the if statement would be false

Reply:
should it error in that case or do we want to update roads with data from failed responses (if there is any)?

Reply:
I think if it has an error there won't be any road data?  Not 100% sure but I believe that is the case

Reply:
ok so it would just error anyways. great